CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Main
 
+* [5085](https://github.com/grafana/loki/pull/5085) **aknuds1**: Upgrade Cortex to e0807c4eb487 and Prometheus to 692a54649ed7
```

Comment:
Just an idea: What about linking the diffs for Cortex and Prometheus between old commit and new commit hash? Guess this could be useful to see the actual changes.

Reply:
@chaudum I can add that.

Reply:
Done.